src/app/content/highlights/components/__snapshots__/EditCard.spec.tsx.snap

```diff
@@ -170,23 +160,6 @@ exports[`EditCard matches snapshot when focused 1`] = `
   data-analytics-region="edit-note"
   data-highlight-card={true}
 >
-  <div
-    color="yellow"
-    mock-color-picker={true}
-    onChange={[Function]}
-    onRemove={[Function]}
-  />
-  <div
-    mock-note={true}
-    note=""
-    onChange={[Function]}
-    onFocus={[Function]}
-    textareaRef={
-      Object {
-        "current": null,
-      }
-    }
-  />
```

Comment:
why are these removed?

Reply:
This looks like the other case: not logged in, so you only get the login confirm part.

Reply:
oh i see, the test names don't make it very clear but i see in the `matches snapshot when focused` that its not setting a user